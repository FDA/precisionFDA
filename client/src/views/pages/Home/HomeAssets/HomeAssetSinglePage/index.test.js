import React from 'react'
import { shallow } from 'enzyme'

import { HomeAssetSinglePage } from './index'


describe('HomeAssetSinglePage test', () => {
  it('should render', () => {
    const component = shallow(<HomeAssetSinglePage />)

    expect(component).toMatchSnapshot()
  })
})
