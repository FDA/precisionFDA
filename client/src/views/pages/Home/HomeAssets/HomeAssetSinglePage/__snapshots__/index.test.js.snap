// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeAssetSinglePage test should render 1`] = `
<HomeLayout>
  <span>
    Asset not found
  </span>
</HomeLayout>
`;
